src/components/booking/time.ts

```diff
@@ -0,0 +1,24 @@
+// time.ts
+import { Dayjs } from 'dayjs';
+
+export const DateFormat = 'YYYY-MM-DD';
```

Comment:
I think we have this constant in constants file

Reply:
yes it has such constant but different format `DATE_FORMAT = 'MM-DD-YYYY';`

Reply:
so, transfer it to constants file too DATE_FORMAT_MONTH_FIRST